interview-questions.md

```diff
@@ -5,56 +5,111 @@ Answer the following questions. First, without external resources. Challenge you
 1. What does CRUD stand for?
 
   Your answer:
+  Create
+  Read
+  Utilize?
+  Delete 
 
   Researched answer:
+  Create
+  Read
+  Update
+  Delete 
 
 
 
 2. What are the 5 HTTP verbs?
 
   Your answer:
+  Push
+  Pull
+  Commit
+  Find
+  Host????
+  
 
   Researched answer:
-
-
+  
+POST, GET, PUT, PATCH, and DELETE
 
 3. When creating a basic (stateless) class component in React, what are the necessary elements needed to render "Hello World" on the page?
 
   Your answer:
 
+
```

Comment:
feel free to reach out to peers if you're having a hard time with a question! we have a pretty supportive bunch here :)